modules/ROOT/pages/architecture.adoc

```diff
@@ -48,6 +48,12 @@ Mule applications run on workers. Multiple replicas of applications can run acro
 
 ** Production environment: 3
 ** Non-production environment: 1
+
+* The maximum number of supported controller nodes is 5.
+
+[NOTE]
```

Comment:
Kim suggests limiting Notes unless absolutely necessary.

Reply:
Good point. I think in this case it's important to point this out, due to the potential problems.